packages/uniforms/src/Bridge.ts

```diff
@@ -84,6 +85,19 @@ export abstract class Bridge {
     );
   }
 
+  /**
+   * Get initial model value recursively.
```

Comment:
```suggestion
   * Get the initial model value walking the tree recursively.
```
Maybe it is nitpicking, but you do not recursively run the `getInitialModel` function.